addon/components/rdf-input-fields/concept-scheme-radio-buttons/edit.js

```diff
@@ -0,0 +1,71 @@
+import { action } from '@ember/object';
+import { guidFor } from '@ember/object/internals';
+
+import SimpleInputFieldComponent from '../simple-value-input-field';
+import { triplesForPath, updateSimpleFormValue} from '@lblod/submission-form-helpers';
+import { SKOS } from '@lblod/submission-form-helpers';
+import rdflib from 'browser-rdflib';
+
+
+function byLabel(a, b) {
+  const textA = a.label.toUpperCase();
+  const textB = b.label.toUpperCase();
+  return (textA < textB) ? -1 : (textA > textB) ? 1 : 0;
+}
+
+export default class RdfInputFieldsconceptSchemeRadioButtonsEditComponent extends SimpleInputFieldComponent {
+  inputId = 'conceptSchemeRadioButtons-' + guidFor(this);
+
```

Comment:
Noticed that you removed the `@tracked` properties? Don't they need to be in a `get` method for them not to be explicitly defined? Or does an `@action` tag also make them tracked?